lib/rails/generators/alchemy/install/templates/elements.yml.tt

```diff
@@ -1,6 +1,6 @@
-# == In this configuration you setup Alchemy´s element layouts.
+# == In this configuration, you set up Alchemy's element layouts.
 #
-# For further informations please see http://guides.alchemy-cms.com/create_elements.html
+# For further information please see http://guides.alchemy-cms.com/edge/elements.html
```

Comment:
Hm, could we have something like `http://guides.alchemy-cms.com/stable/elements.html`? I think the docs should always link to the stable version, if you're adventorous enough to be living on master, you can also find out that on the guides page there's an "edge" link.